### ISSUE TITLE
Invalid squashfs-tools.tar.gz for 8.0 images

### ISSUE THREAD

**Issue by areian** [tatsushid/docker-tinycore#4]
The `squashfs-tools.tar.gz` tarball used in the 8.0 image is invalid, and as such the file is just copied to the image, rather than being unpacked.

```
$ docker run --rm tatsushid/tinycore:8.0-x86_64 ls -l
total 100
drwxr-xr-x    1 root     root           704 Apr 10 23:04 bin
drwxrwxr-x    5 root     root           360 Jun  2 07:43 dev
drwxr-xr-x    1 root     root           454 Jun  2 07:43 etc
drwxrwxr-x    1 root     root             4 Apr 29 03:02 home
-rwxr-xr-x    1 root     root           496 Apr 10 23:04 init
drwxr-xr-x    1 root     root          1104 Apr 10 23:04 lib
lrwxrwxrwx    1 root     root            11 Apr 10 23:04 linuxrc -> bin/busybox
drwxrwxr-x    1 root     root             0 Apr 10 23:04 mnt
drwxrwsr-x    1 root     root           140 Apr 10 23:04 opt
dr-xr-xr-x  248 root     root             0 Jun  2 07:43 proc
drwxrwxr-x    1 root     root            36 Apr 10 23:04 root
drwxrwxr-x    1 root     root             8 Apr 10 23:04 run
drwxr-xr-x    1 root     root           844 Apr 10 23:04 sbin
-rw-r--r--    1 root     root         93806 Apr 29 03:02 squashfs-tools.tar.gz
dr-xr-xr-x   13 root     root             0 Jun  2 07:43 sys
drwxrwxrwt    1 root     root            38 Apr 29 03:02 tmp
drwxr-xr-x    1 root     root            40 Apr 10 23:04 usr
drwxrwxr-x    1 root     root            52 Apr 10 23:04 var
```
Notice the tar.gz file on the 5th line from the bottom.

There are no issues with the file in 7.2

**Comment by inno-steffg** [comment]
+1

**Comment by tatsushid** [comment]
Thanks for reporting the issue. I fixed it at commit e3aadaa.
